qadi_simple.py

```
@@ -327,8 +329,20 @@ async def run_qadi_analysis(
+                llm_provider = None
+                if LLMProvider.GOOGLE in llm_manager.providers:
+                    llm_provider = llm_manager.providers[LLMProvider.GOOGLE]
+                    print("🧬 Semantic evolution operators: ENABLED")
+                else:
+                    print("🧬 Semantic evolution operators: DISABLED (using traditional operators)")
+                
+                # Create genetic algorithm instance with optional LLM provider
+                ga = GeneticAlgorithm(
+                    use_cache=True,
+                    cache_ttl=3600,
+                    llm_provider=llm_provider
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)